components/src/preact/components/error-display.tsx

```diff
@@ -115,5 +125,21 @@ function getDisplayedErrorMessage(error: Error) {
         };
     }
 
+    if (error instanceof InvalidPropsError) {
+        const firstError = error.zodError.errors[0];
+        let message = error.zodError.issues
+            .map((issue) => `Unexpected value for "${issue.path}": ${issue.message}`)
```

Comment:
```suggestion
           .map((issue) => {
                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                // @ts-expect-error
                return `Unexpected value "${JSON.stringify(error.componentProps[issue.path])}" for "${issue.path}": ${issue.message}`;
            })
```
It would be nice to show also the value, which is wrong. I did not find a better solution, than to use the "ts-expect-error".